core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -2414,7 +2414,8 @@ object AdminClientIntegrationTest {
     var result: TopicDescription = null
 
     TestUtils.waitUntilTrue(() => {
-      val topicResult = client.describeTopics(Set(topic).asJava, describeOptions).values.get(topic)
+      val topicResult = client.describeTopics(Set(topic).asJava,
```

Comment:
Seems `getTopicMetadata` is only called at the beginning of `testCreatePartitions`, but from the reported trace it is not thrown from here.